src/main.zig

```diff
@@ -4389,6 +4368,8 @@ pub fn cmdBuild(gpa: Allocator, arena: Allocator, args: []const []const u8) !voi
         var child_argv = std.ArrayList([]const u8).init(arena);
         var reference_trace: ?u32 = null;
         var debug_compile_errors = false;
+        const micros: u32 = @truncate(@as(u64, @bitCast(std.time.microTimestamp())));
+        var seed: []const u8 = try std.fmt.allocPrint(arena, "{}", .{micros});
```

Comment:
This is not how to seed a random number generator. You're involving a bunch of unrelated concepts such as the current time, memory allocation, and formatted string printing. Think about how unnecessarily complicated this is. And then at the end of this process, it didn't even succeed at the main goal, which was to obtain 32 random bits.

Anyway just call `std.crypto.random.int(u32)`.